test/org/sagebionetworks/bridge/dynamodb/DynamoUploadDaoTest.java

```diff
@@ -18,7 +17,6 @@
 import org.springframework.test.context.ContextConfiguration;
 import org.springframework.test.context.junit4.SpringJUnit4ClassRunner;
 
-import org.sagebionetworks.bridge.exceptions.ConcurrentModificationException;
```

Comment:
In a more recent merge, some code was added back to this test that requires that CME. Next check in should remove this removal.